script/test-robots

```diff
@@ -115,10 +109,12 @@ def main(args: argparse.Namespace) -> None:
             with ZipFile(args.archives_dir) as zipfile:
                 zipfile.extractall(archives)
 
-        subprocess_timeout = (
-            EXCESS_SUBPROCESS_TIMEOUT +
-            run_robot_match(run_comp_match, archives, '-', None, None)
-        )
+        start_time = time.time()
+        if not run_robot_match(run_comp_match, archives, '-', None):
+            exit(1)
+
+        match_duration = round(time.time() - start_time)
```

Comment:
What's the purpose of rounding here?

Reply:
That's a leftover from when I was considering rounding up but timeout is actually a float so there is no need to round.